execution/executor-benchmark/src/main.rs

```diff
@@ -6,20 +6,42 @@ use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 struct Opt {
-    #[structopt(long, default_value = "1000000")]
-    num_accounts: usize,
-
-    #[structopt(long, default_value = "1000000")]
-    init_account_balance: u64,
-
     #[structopt(long, default_value = "500")]
     block_size: usize,
 
-    #[structopt(long, default_value = "1000")]
-    num_transfer_blocks: usize,
+    #[structopt(subcommand)]
+    cmd: Command,
+}
+
+#[derive(Debug, StructOpt)]
+enum Command {
+    RunDb {
+        #[structopt(long, parse(from_os_str))]
+        db_dir: PathBuf,
```

Comment:
the rocksdb dir is actually `db_dir/diemdb`, right? To reduce potential confusion, how about let's name this case_dir, suite_dir, data_dir, output_dir etc?